### DISCUSSION TITLE
RM AddOn wie Updaten

### DISCUSSION THREAD

**Discussion by Ricomat** [OpenCCU/OpenCCU#1785] [🇩🇪 Fragen&Antworten] [answered]
Ich hatte bisher Rasberrymatic eigenständig am laufen. Jetzt habe ich das AddOn im HA installiert. Aber wie bekomme ich ein Update drauf? In der Zentralen Wartung gibt ist keine Möglichkeit ein Update auszuwählen?
![keine Update Möglichkeit](https://user-images.githubusercontent.com/49116206/161320645-a35a021b-0b99-4d1e-8665-217011cb2bf2.JPG)

**Comment by jens-maus** [comment, marked as answer]
Die Vorgehensweise ist beim RaspberryMatic CCU HomeAssistant Addon genau wie bei jedem anderen HomeAssistant Addon: Updates werden (sobald es die gibt) in der  jeweiligen HA Addon Konfigurationsseite angezeigt und dann auch dort durchgeführt und dann aktualisiert HomeAssistant eben den gesamten RaspberryMatic CCU Docker Container (nicht anderes ist das bei HA) im Hintergrund inkl Neustart des Addon.

Ein Update durchzuführen ist also im HomeAssistant bzw Docker Fall signifikant anders weil natürlich das Addon/Container sich nicht von innerhalb des Addons/Container aktualisieren kann, sondern eben von ausserhalb - also von der HomeAssistant Oberfläche entsprechend.

**Comment by Ricomat** [comment]
Gut zu wissen! Mein Backup vom aktuellen Standallone Raspberrymatic will es in HA deshalb nicht nehmen. Dann warte ich mal bis in HA ein Update angeboten wird. "Automatische Updates" habe ich mal angehakt. Vielen Dank Jens Maus!

**Comment by Ricomat** [comment]
Wenn man im AddonStore auf die 3 Punkte oben rechts klickt und "Auf Updates prüfen" findet er es. Perfekt!
